### ISSUE TITLE
Quota Exceeded

### ISSUE THREAD

**Issue by ErrorNoInternet** [ayaka14732/ChatGPTAPIFree#5]
Getting this when making a request:

HTTP 429
```
OpenAI API responded with:

{
    "error": {
        "message": "You exceeded your current quota, please check your plan and billing details.",
        "type": "insufficient_quota",
        "param": null,
        "code": null
    }
}
```

**Comment by ErrorNoInternet** [comment]
Seems to be working again now!
It might happen again in a few hours though, I've seen this happen lots of times

**Comment by ayaka14732** [comment]
Yes... At current usage, the API costs $155.72 per day, and I am sadly not able to afford it,

**Comment by owlcloud** [comment]
Listen, can't we implement using a large number of free chatgpt accounts, where 20 requests per hour are given, if I'm not mistaken, and make an array of them with a counter. In case of an error, just switch to the next account. It's an idea, just tell me, do you think it's possible?

**Comment by code-yeongyu** [comment]
> Listen, can't we implement using a large number of free chatgpt accounts, where 20 requests per hour are given, if I'm not mistaken, and make an array of them with a counter. In case of an error, just switch to the next account. It's an idea, just tell me, do you think it's possible?

Guess that means it requires large number of phone number since openai requires it

**Comment by owlcloud** [comment]
Yes, but it is quite feasible and will be free in the future

**Comment by ayaka14732** [comment]
> Listen, can't we implement using a large number of free chatgpt accounts, where 20 requests per hour are given, if I'm not mistaken, and make an array of them with a counter. In case of an error, just switch to the next account. It's an idea, just tell me, do you think it's possible?

This is indeed similar to the current solution: <https://github.com/ayaka14732/ChatGPTAPIFree/blob/8c6bcf2/src/index.js#L66>

**Comment by ayaka14732** [comment]
I have recharged it, and it will continue to work for a while. However, in the long run, the ultimate solution is for skilled individuals to deploy their own backend and share it among a small group of people

**Comment by ayaka14732** [comment]
The free API is not operational at the moment. The current cost for using the API is $155.72 per day, and it seems unfeasible to sustain the API service without a substantial donation.

My ultimate goal is to make ChatGPT more easily accessible to a wider audience, particularly those who lack technical expertise, and enable them to experience the benefits of technological advancement.

To achieve this, I propose developing small-scale API backends for sharing among a small group of people. These backends would be set up by a technically proficient person and then shared among their friends. On the freechatgpt.chat website, people can access the ChatGPT API using these API endpoints.

If many technically proficient people adopt this approach, more people will be able to access ChatGPT.

**Comment by ayaka14732** [comment]
I'm thrilled to share that our free API has received a new lease on life, all thanks to the kindness and generosity of a fellow group member who sponsored our project with 5000 CNY (724 USD)! With this support, we can continue to provide our users with free access to ChatGPT.

To ensure that the API remains stable and secure, I've implemented rate limits through Google Cloud Armor. This will help prevent any abuse of the API, so everyone can continue to benefit from it.

The real magic happens when we all come together to support this project. We need your help to keep this project thriving. You can support us by sponsoring this project as well or, if you have the technical know-how, you can host your own API endpoint and share it with a small group of people.

Together, we can make sure that ChatGPT remains available to all those who need it. Thank you for being a part of our community!

**Comment by hienchu** [comment]
Given every new account on Open AI is given a $18 free trial, it should be possible for people sharing their API keys to the project.

**Comment by saliksik** [comment]
> Given every new account on Open AI is given a $18 free trial, it should be possible for people share their API keys to with the project.

As per the recent updates, newly created accounts will be provided with only $5 worth of free trial credits.

**Comment by hienchu** [comment]
> 

Good to know! $5 is still sizable donation if people willing to contribute

**Comment by renyijiu** [comment]
```bash
curl https://chatgpt-api.shn.hk/v1/ \
  -H 'Content-Type: application/json' \
  -d '{
  "model": "gpt-3.5-turbo",
  "messages": [{"role": "user", "content": "Hello, how are you?"}]
}'
OpenAI API responded:

{
    "error": {
        "message": "You exceeded your current quota, please check your plan and billing details.",
        "type": "insufficient_quota",
        "param": null,
        "code": null
    }
}
```

Looks like it's no longer available

**Comment by hhhh124hhhh** [comment]
Looks like it's no longer available

**Comment by ayaka14732** [comment]
I have increased the limit, and API access should be reinstated shortly. However, we are soon running out of fee. In this case, considering the use of New Bing could be a good idea, especially because it is powered by the more advanced GPT-4 model.

**Comment by manm15** [comment]
Use text-curie-001

A lot cheaper and faster. GPT 3.5 too expensive.

**Comment by czyt** [comment]
I wrote a golang sdk for this project https://github.com/czyt/chatgpt
and it's sorry to see that the project out of service,maybe we can deploy at cloudflare

**Comment by Epsilonche** [comment]
Do the limits apply for each day and then get reset or per minute? 
Trying to see if i can use the api tomorrow for my project (i want to fix incorrect subtitles of a movie).
